test/trento/sap_systems/services/health_summary_service_test.exs

```diff
@@ -34,6 +34,8 @@ defmodule Trento.SapSystems.Services.HealthSummaryServiceTest do
         host_id: a_host_id
       )
 
+      insert(:database, id: database_id)
```

Comment:
I would change the order of things.
Put this insert first, after the host read model, and get the database id from it.
Use this add in the `insert(:sap_system)` and the other `database_instance_without_host`.
This is the correct order of things in real life as well